src/users/models/user.model.ts

```diff
@@ -1,4 +1,4 @@
-export interface User {
+export interface IUser {
   id: string;
   login: string;
   password: string;
```

Comment:
And it should be an entity, but not a model

Reply:
removed, it's useless